### PR TITLE
mirage: adjust name (argv_qubes is gone, this is argv_xen)

### PR DESCRIPTION
**Pull request by hannesm**

### CODE REVIEW

**Comment by djs55** [comment]
Probably worth a rebase against master to see if the CI goes green.

**Comment by djs55** [comment]
Actually I can probably force push a rebase myself easily enough...

**Comment by djs55** [comment]
OK, I've force pushed a rebase. Let's see if the CI is working (in general)

File: lib/mirage.ml [hunk 1]
```diff
@@ -1246,7 +1246,7 @@ let no_argv = impl @@ object
 let argv_xen = impl @@ object
     inherit base_configurable
     method ty = Functoria_app.argv
-    method name = "argv_qubes"
+    method name = "argv_xen"
```
Comment (samoht) [inline lib/mirage.ml]: does it need a change in the package constraints too?

Comment (hannesm) [inline lib/mirage.ml]: it's only the name, not used for any package dependencies..

**Pull request merged**